### PR TITLE
Exploded WallabagRestController into many controllers

### LINKED ISSUE #2503: Explode WallabagRestController into many controllers
Today, we have a big file, `Wallabag\ApiBundle\Controller\WallabagRestController` for all of our API routes.

We need to explode it into many files: 
* EntryRestController
* TagRestController
* ...

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes I hope
| Documentation | no
| Translation   | no
| Fixed tickets | #2503 
| License       | MIT

Fix #2503

### CODE REVIEW

**Comment by nicosomb** [comment]
I wonder if all tests are still executed...

File: src/Wallabag/ApiBundle/Resources/config/routing_rest.yml [hunk 1]
```diff
@@ -1,4 +1,14 @@
-entries:
+api:
   type: rest
   resource:     "WallabagApiBundle:WallabagRest"
   name_prefix:  api_
```
Comment (j0k3r) [inline src/Wallabag/ApiBundle/Resources/config/routing_rest.yml]: I don't think we still need that.
There aren't any action in that file anymore

**Comment by j0k3r** [comment]
If you wonder it means you didn't launch them locally? :flushed:

**Comment by nicosomb** [comment]
All is right 👍  I lost no test.

**Pull request merged**